pages/blog/[slug].tsx

```diff
@@ -26,16 +28,41 @@ export default function BlogPost({ frontMatter, source }: Props) {
 	}, []);
 
 	return (
-		<Flex justifyContent="center" p="6">
-			<AuthorInfo show={showAuthor} data={frontMatter.author} />
-			<Box ml={{ base: '0', lg: '9' }} overflowY="scroll">
-				<BlogHeader metadata={frontMatter} />
-				<Box as="article" id="content" maxW="1000px">
-					<MDXRemote {...source} components={components} />
+		<>
+			<NextSeo
+				title={`${frontMatter.title} - BLACC Blog Post`}
```

Comment:
Remove the " _- BLACC Blog Post_" from the title. You'll more than likely never see that displayed since blog titles tend to have long names.